addons/account/wizard/accrued_orders.py

```diff
@@ -23,10 +23,13 @@ def _get_default_company(self):
         orders = self.env[self._context['active_model']].browse(self._context['active_ids'])
         return orders and orders[0].company_id.id
 
+    def _get_default_journal(self):
+        return self.env['account.journal'].search([('company_id', '=', self.env.company.id), ('type', '=', 'general')], limit=1)
+
     company_id = fields.Many2one('res.company', default=_get_default_company)
     journal_id = fields.Many2one(
         comodel_name='account.journal',
-        compute='_compute_journal_id',
+        default=_get_default_journal,
```

Comment:
```suggestion
        compute='_compute_journal_id", store=True, readonly=False,
```

Reply:
I've tried like this: 
```python
    journal_id = fields.Many2one(
        comodel_name='account.journal',
        compute='_compute_journal_id',
        domain="[('type', '=', 'general'), ('company_id', '=', company_id)]",
        readonly=False,
        store=True,
        required=True,
        check_company=True,
        string='Journal',
    )
```

And it doesn't work. It keep coming back to the `Misc journal`

Reply:
Then just precompute the field
```suggestion
        compute='_compute_journal_id", store=True, readonly=False, precompute=True,
```